ui/selectors/selectors.js

```diff
@@ -144,7 +144,7 @@ import {
 import { getRemoteFeatureFlags } from './remote-feature-flags';
 import { getApprovalRequestsByType } from './approvals';
 
-export const isGlobalNetworkSelectorRemoved = process.env.REMOVE_GNS;
+export const isGlobalNetworkSelectorRemoved = true;
```

Comment:
This enables the feature, we can set to false if needed to render legacy network-filter